City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3899 adjust high school filters more

How to test:

1. Checkout this branch on your Kasko instance.
2. `make drush-cr`
3. Go to add a high school search paragraph to basic page node upper content that has sidebar. Also add one to lower content.
4. Now save the node and check that the filters fit to the space given to them and that selecting emphasis or mission will disable only either emphasis or mission depending on your selection. If line select filter is selected it should not influence emphasis or mission filters.
5. Composer.lock was also updated.
